test/todoAppConstroller.spec.js

```diff
@@ -0,0 +1,52 @@
+describe('todoAppController', function() {
+  beforeEach(module('todoApp'));
+
+  var scope, ctrl;
+
+  beforeEach(inject(function($rootScope, $controller) {
+
+    scope = $rootScope.$new();
+    ctrl = $controller('todoAppController', {
+      $scope: scope
+    });
+  }));
+
+  it('has no tasks on initialisation', function(){
+    expect(scope.todos).toEqual([]);
+  });  
+
+  it('can add a todo item', function(){
+    scope.newTodo = "Task 1";
+    scope.addTodo();
+    expect(scope.todos[0].taskName).toEqual("Task 1");
+  });
+
+  it('can tally all items to do', function(){
+    scope.newTodo = "Task 1";
```

Comment:
You could create a helper method to create new tasks instead of repeating yourself.